### ISSUE TITLE
Running datadriven sequence annotation tests creates files in source tree

### ISSUE THREAD

**Issue by dcolligan** [ga4gh/server#1075]
These files need to be cleaned up after a successful (or failed) test run:

```
$ nosetests tests.datadriven.test_sequenceAnnotations
...
$ git status -s
?? tests/data/datasets/dataset1/sequenceAnnotations/discontinuous.db-shm
?? tests/data/datasets/dataset1/sequenceAnnotations/discontinuous.db-wal
?? tests/data/datasets/dataset1/sequenceAnnotations/gencodeV21Set1.db-shm
?? tests/data/datasets/dataset1/sequenceAnnotations/gencodeV21Set1.db-wal
?? tests/data/datasets/dataset1/sequenceAnnotations/sacCerTest.db-shm
?? tests/data/datasets/dataset1/sequenceAnnotations/sacCerTest.db-wal
?? tests/data/datasets/dataset1/sequenceAnnotations/specialCasesTest.db-shm
?? tests/data/datasets/dataset1/sequenceAnnotations/specialCasesTest.db-wal
```

**Comment by dcolligan** [comment]
These are sqlite generated files... maybe it's better to put them in gitignore

**Comment by dcolligan** [comment]
Fixed in #1078
